src/arch/arm/c_traps.c

```diff
@@ -62,29 +62,54 @@ void VISIBLE NORETURN c_handle_enfp(void)
 }
 #endif /* CONFIG_HAVE_FPU */
 
-static inline void NORETURN c_handle_vm_fault(vm_fault_type_t type)
+void NORETURN vm_fault_slowpath(vm_fault_type_t type)
```

Comment:
this function should still be inline -- the code should not change when `EXCEPTION_FASTPATH` is not enabled.